eth/handler.go

```diff
@@ -372,6 +372,8 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 		if err := msg.Decode(&query); err != nil {
 			return errResp(ErrDecode, "%v: %v", msg, err)
 		}
+		hashMode := query.Origin.Hash != (common.Hash{})
```

Comment:
Why not put this within the `var( )` declaration together with the rest of the declared variables?